TACC/Core-Portal
# Task/WP-520: AppTray should use versionEnabled for list of apps instead of enabled

## Overview

For apps, a specific version could be disabled and another version could be enabled. 
So, the way we were sending requests to `getApps` was no longer valid. 

## Related

* [WP-520](https://tacc-main.atlassian.net/browse/WP-520)

## Changes

I changed the `getPublicApps` function to search only for apps that have an enabled version.

## Testing

1. Navigate to [https://cep.test/core/admin](https://cep.test/core/admin).
2. Click on App Tray Entrys. (Yes, it's spelled that way.)
3. Select any app in the list.
4. Check the version of the app.

## UI

![image](https://github.com/user-attachments/assets/4eb2f9eb-c9a9-4ed9-8284-3b93f1955bc3)

## Notes
